rootfs-builder/rootfs.sh

```diff
@@ -95,6 +99,15 @@ AGENT_INIT          When set to "yes", use ${AGENT_BIN} as init process in place
                     of systemd.
                     Default value: no
 
+RUST_AGENT			When set to "yes", build kata-agent from kata-rust-agent instead of go agent
```

Comment:
I see some indentation issues on the description of the envars.
same on below lines.

Reply:
..looks like tab expand issue, i didn't set expand tab to whitespace

Reply:
fixed